code-scanning/properties/rust-clippy.properties.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "rust-clippy",
+    "creator": "The Rust Programming Language",
```

Comment:
Creator is used if the starter workflow is going to be owned by a ecosystem/organization.
Is that the case here? If no, then don't set this property.

Reply:
How about "The Rust programming language team"? It's owned by https://github.com/rust-lang. All other code scanning workflow meta data have the "creator" property set so I think need to set this.

Reply:
Specifying a creator means that the creator will be responsible for maintenance/review of such starter workflows and not the GitHub Actions team. Please provide a handle who can be contacted to review PR and make critical fixes in the workflow if need be.